crates/wasm-smith/src/core/code_builder.rs

```diff
@@ -2414,6 +2428,12 @@ fn memory_copy_valid(module: &Module, builder: &mut CodeBuilder) -> bool {
         return false;
     }
 
+    // The non-trapping case for memory copy has not yet been implemented,
+    // so we are excluding them for now
+    if !module.config.disallow_traps() {
+        return false;
+    }
```

Comment:
Shouldn't this be 

```rust
if module.config.disallow_traps() {
    return false;
}
```

(That is, no `!`)

?

Reply:
Ah good catch!